class-wc-facebookcommerce.php

```diff
@@ -156,6 +160,7 @@ public function init() {
 				$this->product_sets_sync_handler = new \SkyVerge\WooCommerce\Facebook\ProductSets\Sync();
 				$this->commerce_handler          = new \SkyVerge\WooCommerce\Facebook\Commerce();
 				$this->fb_categories             = new \SkyVerge\WooCommerce\Facebook\Products\FBCategories();
+				$this->background_job_cleanup    = new BackgroundJobCleanup();
```

Comment:
I'd move this into the `JobsRegistry` class.

Reply:
Yep that makes sense, I'm starting to see that as `JobManager` or `JobController`. I can move it there to get behind that.

Reply:
Checked to see if `registry` is an "official" design pattern - https://wiki.c2.com/?RegistryPattern

Reply:
This explains it better - https://www.martinfowler.com/eaaCatalog/registry.html